src/Console/Commands/BassetInstall.php

```diff
@@ -0,0 +1,90 @@
+<?php
+
+namespace Backpack\Basset\Console\Commands;
+
+use Illuminate\Console\Command;
+use Illuminate\Support\Str;
+use Symfony\Component\Process\Process;
+
+/**
+ * Basset Cache command.
+ *
+ * @property object $output
+ */
+class BassetInstall extends Command
+{
+    /**
+     * The name and signature of the console command.
+     *
+     * @var string
+     */
+    protected $signature = 'basset:install';
+
+    /**
+     * The console command description.
+     *
+     * @var string
+     */
+    protected $description = 'Install Backpack Basset.';
+
+    /**
+     * Execute the console command.
+     *
+     * @return void
+     */
+    public function handle(): void
+    {
+        $this->components->info('Installing Basset');
+
+        // create symlink
+        $this->createSymLink();
+
+        // check if artisan storage:link command exists
+        $this->addComposerCommand();
+
+        $this->newLine();
+        $this->info('  Done');
+    }
+
+    /**
+     * Create symlink logic.
+     *
+     * @return void
+     */
+    private function createSymLink(): void
+    {
+        $message = 'Creating symlink';
+
+        if (file_exists(public_path('storage'))) {
+            $this->components->twoColumnDetail($message, '<fg=yellow;options=bold>SKIPPING</>');
```

Comment:
I think this provides a little more clarity as to WHY the directory wasn't created:

```suggestion
            $this->components->twoColumnDetail($message, '<fg=yellow;options=bold>ALREADY EXISTED</>');
```